src/plugins/intel_npu/src/plugin/npuw/serialization.hpp

```diff
@@ -0,0 +1,184 @@
+// Copyright (C) 2024-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <iostream>
+#include <map>
+#include <memory>
+#include <optional>
+#include <string>
+#include <tuple>
+#include <unordered_set>
+#include <vector>
+
+const constexpr char* NPUW_NAME_IDENTIFIER = "NPUW_serialized_";
+const constexpr uint64_t NPUW_SERIALIZATION_INDICATOR = 0x0123456789abcdef;
```

Comment:
let's be more creative.
Also let's define header via a byte array not number.

Reply:
Done